src/lib/y2network/config_writer/sysconfig.rb

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+Yast.import "Routing"
+
+module Y2Network
+  module ConfigWriter
+    class Sysconfig
+      def write(config)
+        routes = config.routes.map { |r| route_to_hash(r) }
+        Yast::Routing.Import("routes" => routes)
+      end
+
+    private
+
+      def route_to_hash(route)
+        hash =
+          if route.default?
+            { "destination" => "-", "netmask" => "-" }
```

Comment:
Hmm, I also have in UI so many times this `"-"`. I am not sure how to handle it, but I do not like it much, that it is spread everywhere.

Reply:
I do not like it either, but for the time being it is how `Yast::Routing` works and this writer basically imports the configuration into `Yast::Routing`.